packages/react/src/checkbox/icons-svg/IconIndeterminate.tsx

```diff
@@ -0,0 +1,16 @@
+import * as styles from "../Checkbox.css";
```

Comment:
If any other component wants to use the icon, should they have checkbox styling? I think icon should be treated as individual and their styling should be inside the component we want to use. Isn't it?

Reply:
This part is just handling the rendering. We can leave this part as an exception

Reply:
I didn't get the comment.

Reply:
Discuss over a call ?

Reply:
I think, we shouldn't have the style in icon, rather than composing the style in checkbox component. Something like 

<Box aschild {...styles.iconIndeterminate({}})}>
   <IconIndeterminate/>
</Box>

Reply:
Got it